tests/transformers/test_numerical.py

```diff
@@ -33,6 +33,194 @@ def test_fit(self):
         assert transformer.null_transformer.fill_value == expect_fill_value
         assert transformer._dtype == expect_dtype
 
+    def test_fit_rounding_none(self):
+        """Test fit rounding parameter with ``None``
```

Comment:
Can the docstrings be expanded a bit to explain inputs, outputs and side effects?

For example, in this case:
```
Input:
- An array with floats rounded to 1 decimal.
Side Effect:
- rounding and rounding_digits continue to be `None`
```